precompile/bank/IBank.sol

```diff
@@ -7,13 +7,17 @@ IBank constant BANK_CONTRACT = IBank(
     BANK_PRECOMPILE_ADDRESS
 );
 
+struct Coin {
+    string denom;
+    uint256 amount;
+}
```

Comment:
Is it difficult to create a types path to get a common Coin type? 
I want to follow the structure of cosmos-sdk as much as possible.

Reply:
[1ab476c](https://github.com/xpladev/xpla/pull/166/commits/1ab476c7a019c31e2fef549ea81f71af1d099cdb)